070-Integrations/024-cloudflare-workers.mdx

```diff
@@ -0,0 +1,268 @@
+---
+title: Get started with Cloudflare and Xata
+navTitle: Cloudflare Workers
+keywords: ['cloudflare', 'full-text search']
+description: Use Cloudflare and Xata
+slug: getting-started/cloudflare
+published: true
+---
+
+You will need a cloudflare account to deploy your application.
+
+The completed [Cloudflare and Xata code](https://github.com/xataio/examples/tree/main/apps/getting-started-cloudflare-workers) for this
+guide is available via the [Xata `examples` repo](https://github.com/xataio/examples) on GitHub.
+
+## Before you begin
+
+Install the Xata CLI:
```

Comment:
Maybe we can just offload this whole section to a link to https://xata-docs-pr-182.vercel.app/docs/getting-started/installation